targets/TARGET_STM/TARGET_STM32F0/TARGET_NUCLEO_F070RB/device/system_stm32f0xx.c

```diff
@@ -450,7 +450,7 @@ uint8_t SetSysClock_PLL_HSI(void)
   RCC_OscInitStruct.LSIState                = RCC_LSI_OFF;
   RCC_OscInitStruct.PLL.PLLState            = RCC_PLL_ON;
   RCC_OscInitStruct.PLL.PLLSource           = RCC_PLLSOURCE_HSI; // HSI div 2
```

Comment:
Hello, could you move the comment to the line you modified, please ?
Otherwise looks good to me.